tensorboard/components/tf_ng_tensorboard/app.component.ts

```diff
@@ -17,7 +17,7 @@ import {Store} from '@ngrx/store';
 import {State} from './core/core.reducers';
 import {coreLoaded} from './core/core.actions';
 
-import * as _typeHackRxjs from 'rxjs';
+/** @typehack */ import * as _typeHackRxjs from 'rxjs';
```

Comment:
I'm not familiar with this annotation. Can you explain, mainly for my edification?

Reply:
These hacks are not required and, in fact, it gets in the way of syncing. For more detail, I can talk to you offline.

Reply:
The `_typeHackRxjs` isn't used AFAICT. Can it be simplified as

```
/** @typehack */ import 'rxjs';
```
?

Reply:
Nope, I confess I do not understand the nature of the TSC error but it does not like that.

Reply:
OK. Got it. Can you document this for other developers so they'll know what's going on here?

Reply:
It was documented in some places in a GitHub issue that I filed. I am not sure if there is a value in repeating this link to GitHub over and over.